srtcore/api.cpp

```diff
@@ -280,9 +282,14 @@ int srt::CUDTUnited::cleanup()
     // executing this procedure.
     ScopedLock gcinit(m_InitLock);
 
-    if (--m_iInstanceCount > 0)
-        return 0;
+    if (!force)
+    {
+        if (m_iInstanceCount == 0 || --m_iInstanceCount > 0)
+            // [reinstate in 1.6.0] return m_iInstanceCount;
+            return 0;
+    }
 
+    // Cleanup done before already
     if (!m_bGCStatus)
         return 0;
```

Comment:
So `force` will have no effect if cleanup was done already. It only forces setting instance count to zero...

Reply:
This `force` is true when calling from destructor. Its role is to prevent a deferred cleanup.